src/app/core/services/fss-search-filter.service.ts

```diff
@@ -77,4 +77,20 @@ export class FssSearchFilterService {
     return operatorType
   }
 
+  getFilterExpressionForSimplifiedSearch(fssSearchFilter: string): string {
+    let searchKeywords = fssSearchFilter.split(" ");
+     
+    let filterExpression = "";
+    for (var i in searchKeywords) {
+      if (searchKeywords[i] !== "") {
+        var searchKeyword = searchKeywords[i].replace(/'/g, "''");
```

Comment:
Please use `let` here
```suggestion
        let searchKeyword = searchKeywords[i].replace(/'/g, "''");
```

Reply:
resolved